ord_schema/orm/structure_test.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2022 Open Reaction Database Project Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for ord_schema.orm.structure."""
+import pytest
+from sqlalchemy import select
+from ord_schema.orm.mappers import Compound, Reaction, ReactionInput
+from ord_schema.orm.structure import FingerprintType, Structure
+
+
+def test_tanimoto_operator(test_session):
```

Comment:
Real meat of the new commit is in this file.  Once I stand up a version of the database with data internally I think I'll put some time into jupyter notebook examples using this functionality.  Hard to do beyond a toy example with the postgres testing, but it still validates as expected.

```python
test = 'test_reaction-0'
query = (
        select(Reaction)
        .join(ReactionInput)
        .join(Compound)
        .join(Structure)
        .where(Reaction.reaction_id == test[0]['Reaction'].reaction_id)
    )
results_new = test_session.execute(query).fetchall()
results_new[0]['Reaction'].id
```